egs/wsj/s5/steps/libs/nnet3/xconfig/basic_layers.py

```diff
@@ -999,7 +999,7 @@ def set_default_configs(self):
                        'dim': -1,
                        'cepstral-lifter': 22.0,
                        'affine-transform-file': '',
-                       'write-init-config': True}
+                       'write-init-config': False}
```

Comment:
Actcually I'm unclear why this is writing anything to init.config at all in this case, since there is nothing to estimate.
@vimalmanohar, what was the thinking here?